specification/synapse/resource-manager/Microsoft.Synapse/stable/2023-05-01/listUsage.json

```diff
@@ -0,0 +1,183 @@
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/usages": {
+      "get": {
+        "operationId": "Operations_GetWorkspaceQuotaAndUsage",
+        "summary": "Gets the current usage and quota of a workspace.",
+        "description": "Gets the current usage and quota of a workspace.",
+        "tags": [
+          "Usages"
+        ],
+        "deprecated": false,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../common/v1/types.json#/parameters/WorkspaceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "schema": {
+              "$ref": "#/definitions/ListUsagesResult"
+            },
+            "description": ""
+          },
+          "default": {
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            },
+            "description": ""
+          }
+        },
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-examples": {
+          "Subscription usage in given location": {
+            "$ref": "./examples/WorkspaceUsageQuotaGet.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "UsageName": {
+      "type": "object",
+      "description": "Name of the usage metrics",
+      "properties": {
+        "value": {
+          "type": "string",
+          "description": "The name of the usage metric."
+        },
+        "localizedValue": {
+          "type": "string",
+          "description": "The localized name of the usage metric."
+        }
+      }
+    },
+    "Usage": {
+      "type": "object",
+      "description": "Resource usage and limit.",
+      "properties": {
+        "unit": {
+          "type": "string",
+          "description": "The unit of usage measurement."
+        },
+        "currentValue": {
+          "type": "number",
+          "format": "double",
+          "description": "The current usage of the resource."
+        },
+        "limit": {
+          "type": "number",
+          "format": "double",
+          "description": "The maximum permitted usage of the resource."
+        },
+        "name": {
+          "$ref": "#/definitions/UsageName",
+          "description": "The name of the type of usage."
+        }
+      },
+      "required": [
+        "unit",
+        "currentValue",
+        "limit",
+        "name"
+      ]
+    },
+    "ListUsagesResult": {
```

Comment:
>ListUsagesResult [](http://example.com/codeflow?start=5&length=16)

Can u move this to a common type definition for whole of ARM please?

Reply:
Done